client/store/modules/SettingStore.js

```diff
@@ -0,0 +1,34 @@
+import { SETTINGS_PANEL_TOGGLE } from '../StaticActions';
+import { SETTINGS_PANEL_SET_OPEN, SETTINGS_PANEL_SET_CLOSED } from '../StaticMutations';
+
+const state = {
+    settingsPanelOpen: false,
+};
+
+const getters = {};
+
+const actions = {
+    [SETTINGS_PANEL_TOGGLE]({ commit, state }) {
+        console.log('Toggling the settings panel.');
+        commit(state.settingsPanelOpen ? SETTINGS_PANEL_SET_CLOSED : SETTINGS_PANEL_SET_OPEN);
+    },
+};
+
+const mutations = {
+    [SETTINGS_PANEL_SET_OPEN](state) {
+        console.log('Setting settings panel open.');
```

Comment:
"Opening settings panel" is more readible to me :)

Reply:
Should get rid of this testing console log anyway, there are store tracebacks anyway